pkg/install/install.go

```diff
@@ -100,12 +101,51 @@ func DefaultImage() string {
 	return image
 }
 
+func validMailAddress(address string) (string, bool) {
+	addr, err := mail.ParseAddress(address)
+	if err != nil {
+		return "", false
+	}
+	return addr.Address, true
+}
+
 func Install(ctx context.Context, image string, opts *Options) error {
 	// I don't want these errors on the screen. Probably a better way to do this.
 	klog.SetOutput(io.Discard)
 	klogv2.SetOutput(io.Discard)
 	utilruntime.ErrorHandlers = nil
 
+	// Require E-Mail address when using Let's Encrypt production
+	if opts.Config.LetsEncrypt != nil && *opts.Config.LetsEncrypt == "enabled" {
+		if opts.Config.LetsEncryptTOSAgree == nil || !*opts.Config.LetsEncryptTOSAgree {
+			ok, err := pterm.DefaultInteractiveConfirm.Show("You are choosing to use Let's Encrypt production environment for TLS certificates. To do so, you must agree to their Terms of Service: https://community.letsencrypt.org/tos\nTip: use --lets-encrypt-tos-agree to skip this prompt\nDo you agree to Let's Encrypt TOS?")
```

Comment:
There is another thing I don't like about this: as soon as you enter 'y', it continues. It doesnt wait for you to hit enter. I think it should. Our other yes/no prompts (most obvious when you do `acorn uninstall -a`) wait until you hit enter.

This caused me to stumble because I hit y and then enter, but enter actually applied to the next prompt, which was for my email and it caused be to submit a blank email